### PR TITLE
Expanding the health checks doc

### LINKED ISSUE #293: Expand doc on Management Portal health checks
There is potentially information in https://github.com/vmware/admiral/issues/115 that would be useful to add to https://vmware.github.io/vic-product/assets/files/html/1.1/vic_dev_ops/configuring_health_checks.html.

### PR DESCRIPTION
**Pull request by alextopuzov**
Fixes #293

### CODE REVIEW

File: docs/user_doc/vic_app_dev/provision_containers_portal.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ When you create containers from the Containers page in the management portal, yo
 	- Anti-affinity rules
 - Custom environment variables
```
Comment (stuclem) [inline docs/user_doc/vic_app_dev/provision_containers_portal.md]: Can we add something about the env variables that you can set?

File: docs/user_doc/vic_app_dev/provision_containers_portal.md [hunk 2]
```diff
@@ -27,6 +27,9 @@ When you create containers from the Containers page in the management portal, yo
...
 - Health checks
```
Comment (stuclem) [inline docs/user_doc/vic_app_dev/provision_containers_portal.md]: Maybe mention that these are regular docker health checks

Comment (alextopuzov) [reply docs/user_doc/vic_app_dev/provision_containers_portal.md]: From what i've seen here https://github.com/vmware/admiral/issues/115 they are not docker health checks but custom Admiral ones.

File: docs/user_doc/vic_app_dev/provision_containers_portal.md (hunk 1 above)
Comment (alextopuzov) [reply docs/user_doc/vic_app_dev/provision_containers_portal.md]: Like these? https://docs.docker.com/compose/environment-variables/

File: docs/user_doc/vic_app_dev/provision_containers_portal.md (hunk 2 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/provision_containers_portal.md]: OK, then all the more reason to mention it then!

File: docs/user_doc/vic_app_dev/provision_containers_portal.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/provision_containers_portal.md]: Yep, exactly

Comment (alextopuzov) [reply docs/user_doc/vic_app_dev/provision_containers_portal.md]: Added.

File: docs/user_doc/vic_app_dev/provision_containers_portal.md (hunk 2 above)
Comment (alextopuzov) [reply docs/user_doc/vic_app_dev/provision_containers_portal.md]: It's there now.

**Pull request merged**